motherduck/results/result.json

```diff
@@ -1,57 +1,57 @@
-[0.24364612391218543,0.325953362043947,0.3064295807853341],
-[15.391365340910852,0.8924880139529705,0.8800020259805024],
-[1.0099374479614198,0.7060354948043823,0.7063306840136647],
-[8.954986380413175,4.095173366833478,3.279917892999947],
-[23.613256133161485,19.25096853263676,23.278139277826995],
-[1.5025351103395224,0.7977945390157402,0.6170151601545513],
-[0.46431142510846257,0.4349544760771096,0.36817489471286535],
-[0.5523030371405184,0.5110767548903823,0.3649540198966861],
-[4.98280710587278,2.0472186598926783,2.6902666091918945],
-[6.719612340908498,4.538661384955049,4.369533614721149],
-[3.358773229178041,3.1345412940718234,3.100874589756131],
-[2.000370934139937,1.3284917967393994,1.2289274740032852],
-[3.7089375513605773,2.681138473097235,1.91789041319862],
-[7.777549963910133,7.958697326015681,7.435122138820589],
-[9.89957385417074,6.938831991981715,5.757995248306543],
-[5.393568014726043,5.632281670346856,5.551704770885408],
-[1.4525640471838415,0.8954971083439887,0.9103033309802413],
-[0.22383009875193238,0.2152935261838138,0.21353887394070625],
-[0.2124440912157297,0.20737810991704464,0.20923770079389215],
-[0.2175677828490734,0.23641109932214022,0.22010038886219263],
-[0.24250942608341575,0.21723396237939596,0.2134688631631434],
-[0.21473077218979597,0.2085639163851738,0.251109620090574],
-[0.21749612782150507,0.24898552894592285,0.20936914114281535],
-[0.2193519319407642,0.2155164610594511,0.20863541215658188]
+[0.022321860000374727,0.012934670001413906,0.012823413999285549],
+[0.024380385999393184,0.012925036000524415,0.012814411998988362],
+[0.038363479001418455,0.015576638001221,0.01562748600190389],
+[0.03155873300056555,0.021764302997326013,0.02121602799888933],
+[0.17027187499843421,0.14739435199953732,0.14690826999867568],
+[0.20644352700037416,0.19285383399983402,0.18173918799948297],
+[0.02371823299836251,0.012589622001542011,0.012246084999787854],
+[0.04112412900212803,0.016385651997552486,0.01679801299906103],
+[0.1886740549998649,0.1850254379969556,0.18047275500066462],
+[0.25464338600068004,0.24228513200068846,0.24277507700026035],
+[0.06307598200146458,0.06222396100201877,0.06118656300168368],
+[0.0769712619985512,0.06881667799825664,0.07263678299932508],
+[0.17805651500020758,0.17287376500098617,0.17574450200118008],
+[0.35226519700154313,0.3180267339994316,0.32635465799830854],
+[0.19063701599952765,0.18687298700024257,0.18603799500124296],
+[0.21010683900021832,0.1866715190008108,0.198137998002494],
+[0.38373803200011025,0.37373249000302167,0.371793443999195],
+[0.3542730239969387,0.33429408599840826,0.33363083899894264],
+[0.6366427199973259,0.6410418160012341,0.6345498260016029],
+[0.05052374699880602,0.014082836998568382,0.014972674998716684],
+[0.3299962970013439,0.30164447399874916,0.3019338140002219],
+[0.2213447880021704,0.221652765001636,0.22077769899988198],
+[0.3728656390012475,0.3468815829983214,0.34391914299703785],
+[1.2559020400003647,1.2022961519978708,1.2722901870001806],
```

Comment:
@jtigani Just for my understanding: Cold (1st) and hot (2nd/3rd) runtimes are quite similar in most of the new measurements. Given that the `hits` table is quite big (75GB TSV), this can mean a few things:
- the servers are equipped with lots of DRAM and the data remains essentially in memory after the first load. I am speculating of course but DRAM is quite costly so I'd deem that unlikely.
- the data is compressed so well that it fits into the main memory of a smaller server
- scans are avoided using skipping techniques. I [recently tried](https://github.com/ClickHouse/ClickHouse/issues/70605#issuecomment-2457022442) min-max (SMA/zonemap) indexes recently on ClickBench but it did not make a difference. (but of course, there other skipping techniques exist, so this is not representative).

No need to reveal business secrets :) , I just wanted to make sure that the measurements contain no accidental mistake. Thanks!